docs/src/documentation/03-components/01-action/button/03-accessibility.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: Accessibility
+redirect_from:
+  - /components/button/accessibility/
+---
+
+# Accessibility
+
+## Button
+
+The Button component has been designed with accessibility in mind.
+
+The following props provide additional information to screen readers:
```

Comment:
I moved `accessibility` section from `React tab` and kept the text content as it was.